### ISSUE TITLE
Questions about the learning strategy of mobilenetv2-openpose

### ISSUE THREAD

**Issue by af-yi** [michalfaber/tensorflow_Realtime_Multi-Person_Pose_Estimation#28]
Excuse me， I want to retrain the model with three refinement stage, but I'm not sure how to set the training strategy. 
So, I want know the weights_path = "best_pose_mobilenet_model/weights.best.mobilenet.h5"  this mobilenetv2-openpose 2stage model trained by train_custom_loop_mobilenet.py?  batch-size=10,max-epochs=100? AND no mask?
